content/pages/constitution.md

```diff
@@ -0,0 +1,313 @@
+Author: Fatema Boxwala
+Title: Constitution
+hidden: true
+
+# Structure of Organization #
+
+The Women in Computer Science Undergraduate Subcomittee is a
+subcommittee of Women in Computer Science, a standing committee of the
+Cheriton School of Computer Science. For the remainder of this document,
+this standing committee will be referred to as the "WiCS Committee". All
+references to: "the undergraduate committee", "the committee", and
+"committee members" refer to the Undergraduate Subcommittee unless
```

Comment:
Undergraduate *Committee*

Just for historical reference, the committee was referred to as the "subcommittee" for a portion of F14 (first written reference of this was in the MathSoc MOU which I got signed in Aug. '14). Jo suggested we change it to "committee" and so that's been the official name ever since.